package.json

```diff
@@ -185,24 +189,19 @@
             ],
             "editor/title": [
                 {
-                    "command": "lean.infoView.toggleMode",
-                    "when": "resourceScheme == lean-info",
+                    "command": "lean.displayList",
+                    "when": "editorLangId == lean",
```

Comment:
Can we show the buttons on top of the info view as well?  At the moment they vanish as soon as soon as you focus the info view (e.g. if you select something).